### PR TITLE
CONTRIBUTING.md Overhaul

### PR DESCRIPTION
**Pull request by Lorwp**
> **FOR THE HONKMOTHER'S SAKE, DO NOT MERGE**
> I'm starting to add some more documentation on Contributing to the repo, any feedback is appreciated, but it will be a while before this merges..

### CODE REVIEW

**Comment by BlueNexus** [comment]
Slight extra bracket issue in the second file.

**Comment by Lorwp** [comment]
@BlueNexus Per line comments would be Appreciated :)

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <!-- Change Title to 'NSS Phoenix' If Kaedwulf rules over Station -->
 # Aphelion Project
-![Travis Build Status](https://travis-ci.org/sshado/Aphelion-Code.svg)
+[![Travis Build Status](https://travis-ci.org/sshado/Aphelion-Code.svg)
```
Comment (BlueNexus) [inline README.md:3]: Extra bracket here, for some reason.

Comment (Lorwp) [inline README.md:3]: @BlueNexus Formatting Breaks if that bracket is removed

**Comment by Spacemanspark** [comment]
RIP

**Comment by Lorwp** [comment]
Indeed

**Pull request closed without merging**